content/updates/2021-03/5-Events-API-logs.md

```diff
@@ -0,0 +1,25 @@
+---
+pim_announcement_alt_img: "Events API log screen"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Browse your Events API logs
+::: meta-data type="New" features="Connectivity" available="late March" in="EE,GE" link-to-doc="TODO"
```

Comment:
Doc link: https://help.akeneo.com/pim/serenity/articles/manage-event-subscription.html#debug

Reply:
```suggestion
::: meta-data type="New" features="Connectivity" available="late March" in="EE,GE" link-to-doc="../articles/manage-event-subscription.html#debug"
```

Reply:
I'll merge this once the doc will be updated with content about the whole feature (and not just the download button).